testng-core/src/test/java/test/configuration/sample/SuiteTestSample.java

```diff
@@ -1,20 +1,15 @@
-package test.configuration;
+package test.configuration.sample;
 
 import org.testng.annotations.BeforeSuite;
 import org.testng.annotations.Test;
+import test.configuration.BaseSuiteTest;
 
-public class SuiteTest extends BaseSuiteTest {
+public class SuiteTestSample extends BaseSuiteTest {
```

Comment:
A Sample must not extend `BaseSuiteTest `

Reply:
BaseSuiteTest moved to samples